R/rowClust.R

```diff
@@ -0,0 +1,41 @@
+# REQUIRED PACKAGES: "fastcluster", "stats", "coop"
+rowClust <- function(tse,
+                     abund_values = "counts",
+                     metric = "spearman", # c("pearson", "kendall", "spearman" || 
+                                          #    "cosine" ||
+                                          #    "euclidean", "maximum", "manhattan", "canberra", "binary", "minkowski")
+                     method = "ward.D2", # c("single", "complete", "average", "mcquitty", "ward.D", "ward.D2", "centroid","median")
+                     name = "clust_id",
+                     k = NULL, # number of clusters
+                     h = NULL, # height of cut-off
+                     ... # arguments passed to stats::dist (mainly for the p arg of "minkowski")
+                     ) {
+  # Getting Assay data
+  if(abund_values %in% assayNames(tse))
+    x <- assay(tse, abund_values)
+  else
+    stop(paste0("Assay ",abund_values," not Found!"))
+  
+  # Calculating distance
+  if (metric %in% c("pearson", "kendall", "spearman"))
```

Comment:
I think `metric` is called `method` in `dist`. 

How about naming methods as `clustering_method` and `dissimilarity_method`?

Reply:
sure